core/shared/src/main/scala/org/typelevel/log4cats/extras/WriterLogger.scala

```diff
@@ -21,6 +21,16 @@ import cats.data._
 import cats.syntax.all._
 import org.typelevel.log4cats._
 
+/**
+ * A `SelfAwareLogger` implemented using `cats.data.Writer`.
+ *
+ * >>> WARNING: READ BEFORE USAGE! <<<
+ * https://github.com/typelevel/log4cats/blob/main/core/shared/src/main/scala/org/typelevel/log4cats/extras/README.md
```

Comment:
Wouldn't it be better to put these warnings on the microsite and leave here the link on it?

Reply:
IIRC neither of these classes is on the microsite, so putting the warning there might draw people to them, rather than nudge them towards less niche classes.